lldb/source/Symbol/SymbolContext.cpp

```diff
@@ -163,6 +176,14 @@ bool SymbolContext::DumpStopContext(Stream *s, ExecutionContextScope *exe_scope,
       dumped_something = true;
       if (symbol->GetType() == eSymbolTypeTrampoline)
         s->PutCString("symbol stub for: ");
+
+      // Similar here, Using DumpName if the function is called by regex symbol
```

Comment:
Just an FYI, readers will not necessarily be reading code in a straight line. So relative references like "similar here" are mostly useless unless very close to each other.

Also we'll want to remove these comments eventually but they are fine to keep around while you iterate, if it helps you.

Reply:
Removed the comments.